docs/whats-new/dotnet-docs-mod2.md

```diff
@@ -1,107 +1,140 @@
-- [DirectoryServices package no longer references System.Security.Permissions](../core/compatibility/extensions/8.0/directoryservices-package.md)
-- [Trimming may not be used with .NET standard or .NET framework](../core/compatibility/sdk/8.0/trimming-unsupported-targetframework.md)
-- ['dotnet restore' produces security vulnerability warnings](../core/compatibility/sdk/8.0/dotnet-restore-audit.md)
-- [.NET can't be installed to custom location](../core/compatibility/sdk/6.0/install-location.md)
-- [Boolean-backed enum type support removed](../core/compatibility/core-libraries/8.0/bool-backed-enum.md)
-- [Binding config to dictionary extends values](../core/compatibility/extensions/7.0/config-bind-dictionary.md)
-- ['ca-certificates' and 'krb5-libs' packages removed from Alpine images](../core/compatibility/containers/8.0/krb5-libs-package.md)
-- [Security token events return a JsonWebToken](../core/compatibility/aspnet-core/8.0/securitytoken-events.md)
-- [TrimMode defaults to full for Web SDK projects](../core/compatibility/aspnet-core/8.0/trimmode-full.md)
-
-## .NET Framework
+- [Enumerable.Sum throws new OverflowException for some inputs](../core/compatibility/core-libraries/8.0/enumerable-sum.md)
+- [ProcessStartInfo.WindowsStyle honored when UseShellExecute is false](../core/compatibility/core-libraries/8.0/processstartinfo-windowstyle.md)
+- [PublishedTrimmed projects fail reflection-based serialization](../core/compatibility/serialization/8.0/publishtrimmed.md)
+- [Source Link included in the .NET SDK](../core/compatibility/sdk/8.0/source-link.md)
+- [Unlisted packages not installed by default for .NET tools](../core/compatibility/sdk/8.0/unlisted-versions.md)
+- [Implicit `using` for System.Net.Http no longer added](../core/compatibility/sdk/8.0/implicit-global-using-netfx.md)
+- [SendFile throws NotSupportedException for connectionless sockets](../core/compatibility/networking/8.0/sendfile-connectionless.md)
+- [System.Windows.Extensions doesn't reference System.Drawing.Common](../core/compatibility/windows-forms/8.0/extensions-package-deps.md)
+- [JSFunctionBinding implicit public default constructor removed](../core/compatibility/interop/8.0/jsfunctionbinding-constructor.md)
+- [System.Formats.Cbor DateTimeOffset formatting change](../core/compatibility/extensions/8.0/cbor-datetime.md)
 
 ### Updated articles
 
-- [Security Protocols version 1.0](../framework/wcf/feature-details/security-protocols-version-1-0.md) - Fix build suggestions
-- [\<defaultProxy> element (network settings)](../framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md) - Add link to DefaultProxy property
+- [Breaking changes in .NET 6](../core/compatibility/6.0.md) - Remove "introduced version" column
 
 ## .NET fundamentals
 
 ### New articles
 
-- [Logging guidance for .NET library authors](../core/extensions/logging-library-authors.md)
+- [Microsoft.DotNet.ApiCompat.Tool global tool](../fundamentals/apicompat/global-tool.md)
```

Comment:
Can/should we order these alphabetically?

Reply:
Good idea. Captured as docs-tools#276